### PR TITLE
Bind close button for attachments

### LINKED ISSUE #203: Cannot close a pdf opened from an attachment inside pdf documents
## Steps to reproduce

1-  Open attachments inside pdf document (it is opened in a new tab, intended behaviour?)
2 - Try to close the attachment using the close cross Icon of the files pdfviewer.

## Expected behavior

The file is close and it is possible to go back to files view

## Current behavior

Close icon does not trigger any action.

Screen Shot 2019-12-10 at 12 36 10
Screen Shot 2019-12-10 at 12 36 36

  @HanaGemela please triage (cc @VicDeo @micbar jfyi)

### PR DESCRIPTION
**Pull request by VicDeo**
### Steps to reproduce
1- Open attachments inside pdf document
2 - Try to close the attachment using the close cross Icon of the files pdfviewer.

Fixes #203 
includes https://github.com/owncloud/files_pdfviewer/pull/200

### CODE REVIEW

**Comment by VicDeo** [comment]
PHPUnit failure is unrelated to the JS changes.
Caused by newer PHPUnit in core.

**Comment by micbar** [comment]
@VicDeo Yes, we updated PHPunit.

File: js/workersrc.js [hunk 1]
```diff
@@ -13,6 +13,14 @@ function redirectIfNotDisplayedInFrame () {
 redirectIfNotDisplayedInFrame();
 
 function deferredViewerConfig() {
+	if (location.href.indexOf('?file=blob') !== -1) {
+		document.getElementById('secondaryToolbarClose').addEventListener(
+			'click',
+			function(){
```
Comment (jvillafanez) [inline js/workersrc.js]: include an space? `function() {`

Comment (VicDeo) [reply js/workersrc.js]: thanks, updated

**Comment by micbar** [comment]
@VicDeo I know the tests are broken because of the upgrade, but we need to fix them.

@phil-davis Any ideas, examples from other repos?

**Comment by VicDeo** [comment]
@micbar as I wrote it's completely unrelated. 
It shuld be easy but I have no room for fixing PHP tests now unless it has higher prio than other tasks.  :/

**Comment by micbar** [comment]
> @micbar as I wrote it's completely unrelated.
> It should be easy but I have no room for fixing PHP tests now unless it has higher prio than other tasks. :/

Thats fine.

**Comment by phil-davis** [comment]
This will need a rebase - PR #207 fixed up the unit test file naming...

**Comment by VicDeo** [comment]
@phil-davis the base is not master here, so I just updated a declaration for `setUp`

**Comment by phil-davis** [comment]
OK - I had not noticed that

**Pull request merged**